src/utils/create-excerpt.js

```diff
@@ -0,0 +1,18 @@
+import MarkdownIt from "markdown-it";
+import { convert } from "html-to-text";
+const parser = new MarkdownIt();
+
+export const createExcerpt = (body) => {
+  const html = parser.render(body);
+  const options = {
+    wordwrap: null,
+    selectors: [
+      { selector: "a", options: { ignoreHref: true } },
+      { selector: "img", format: "skip" },
+      { selector: "figure", format: "skip" },
+    ],
+  };
+  const text = convert(html, options);
+  const distilled = convert(text, options);
```

Comment:
Interestingly, the first pass of the parser was unable to handle additional tags within <p> tags, because that content ended up being escaped, so the parser parsed the escaped characters back to tags, hence one more round of distilling was required to also parse the parsed tags.
乁⁠(⁠ ⁠•⁠_⁠•⁠ ⁠)⁠ㄏ